tests/test_tf_helper.py

```diff
@@ -0,0 +1,20 @@
+import tensorflow as tf
+
+from benchopt.utils.safe_import import set_benchmark
+
+set_benchmark('./')
```

Comment:
A safer way to set the benchmark is usually to use a relative location wrt `__file__`, so that it is independent of the runnign dir :
```suggestion
from pathlib import Path

set_benchmark(Path(__file__).parents[-2])
```